CODE_OF_CONDUCT.md

```diff
@@ -68,9 +68,9 @@ members of the project's leadership.
 ## Attribution
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+available at <https://www.contributor-covenant.org/version/1/4/code-of-conduct.html>
```

Comment:
I haven't seen this syntax before for links? Is this documented in the CommonMark spec somewhere? What's the benefit?

Reply:
This was added to make sure the raw links stay as intented.